Gems/Atom/Feature/Common/Assets/Shaders/SkyAtmosphere/SkyRayMarching.azsl

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+ // Based on https://github.com/sebh/UnrealEngineSkyAtmosphere by Sébastien Hillaire
+
+#define ENABLE_ATMOSPHERE_SHADOWS 1
```

Comment:
Is this macro required? it is also declared 3 times

Reply:
It was added to avoid shader compilation issues that occurred because all three passes (raymarching, transmittence, skyview) use the same integration function so some needed the o_shadowsEnabled on and some need it off.  I can probably  think of a way to ensure I only turn that shader variant on for the raymarching shader.